<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlNew</class>
 <widget class="QWidget" name="ControlNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>112</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验</string>
  </property>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStartStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbSelection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>实验选择</string>
   </property>
   <widget class="QRadioButton" name="rbExp1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实验一</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbExp2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实验二</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbExercise">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>88</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>练习模式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
